js/livecoin.js

```diff
@@ -343,12 +343,14 @@ module.exports = class livecoin extends Exchange {
         await this.loadMarkets ();
         let method = 'privatePostExchange' + this.capitalize (side) + type;
         let market = this.market (symbol);
+        let precisionPrice = market['precision']['price'];
+        let precisionAmont = market['precision']['amount'];
         let order = {
-            'quantity': amount,
+            'quantity': amount.toFixed(precisionAmont),
```

Comment:
Let's remove L346 and L347
and in L349 let's write:
```
let order = {
     'quantity': this.amountToPrecision (symbol, amount),
     'currencyPair': market['id'],
};
```

This would be a cross-compatible way to do it )

And in L353 bellow let's write:

```
order['price'] = this.priceToPrecision (symbol, price);
```

This would do the trick for us )

Reply:
Done